docs/installation/autoware/source-installation.md

```diff
@@ -20,30 +20,166 @@ sudo apt-get -y install git
    cd autoware
    ```
 
-2. Install the dependencies.
+2. You can install the dependencies either manually or using the provided Ansible script.
 
-   We use [Ansible](https://www.ansible.com/) to simplify the steps.
-   Please see the Ansible playbooks and roles if you want to know exactly what packages are installed.
+### Installing dependencies manually
 
-   > Note: Before installing NVIDIA libraries, confirm and agree with the licenses.
+Install ROS2:
 
-   - [CUDA](https://docs.nvidia.com/cuda/eula/index.html)
-   - [cuDNN](https://docs.nvidia.com/deeplearning/cudnn/sla/index.html)
-   - [TensorRT](https://docs.nvidia.com/deeplearning/tensorrt/sla/index.html)
```

Comment:
Could you please do not remove caution? :pray:

Reply:
@xmfcx It seems the indents disappeared.
![image](https://user-images.githubusercontent.com/31987104/161030027-fcb9f0dd-0661-4f7e-aebb-5dad7761feec.png)

It's okay for now, but if you add the section `3. ` after this, then I guess it will break `markdownlint`.